site/docs/master/plugins/get-started.md

```diff
@@ -49,3 +49,14 @@ This pod has both a Configuration and Status.
 ### Items
 
 A plugin with support for `PrinterItems` allow adding a `FlexLayoutItem` consisting of a width and a view component.
+
+## Uninstall
+
+Plugins can be removed by deleting the plugin binary from `~/.config/octant/plugins`. An example of deleting a plugin is shown below 
+where `octant-sample-plugin` is the plugin that will uninstalled:
+
+```
+rm ~/.config/octant/plugins/octant-sample-plugin
+```
+
+After deleting the plugin binary, you should no longer see the plugin available as part of the Octant dashboard. 
```

Comment:
A restart of Octant is current required to stop the plugin proccess, even if the binary has been deleted.

Reply:
Ah, good catch. I see how that isn't communicated clearly. Let me add it.

Reply:
Yes. That is correct. If you stop the plugin process, Octant will restart it.